pkg/qase/README.md

```diff
@@ -0,0 +1,19 @@
+#### Qase Reporter Package
+- Required vars to export locally or add on .env file:
+```
+export REPORT_TO_QASE=true
+export QASE_AUTOMATION_TOKEN=your_token
```

Comment:
So is it `QASE_AUTOMATION_TOKEN` or `QASE_API_TOKEN`?

Reply:
here is QASE_AUTOMATION_TOKEN , cause we inhereted that from rancher....on GH actions its already QASE_API_TOKEN